src/platform/Makefile.am

```diff
@@ -233,6 +233,44 @@ libDeviceLayer_a_SOURCES               += \
 
 endif #CONFIG_BLE_PLATFORM_BLUEZ
 
+Linux/gen/DBusWpa.h: Linux/dbus/wpa.xml
+	gdbus-codegen --header --output $@ --c-namespace Wpa $<
+
+Linux/gen/DBusWpaInterface.h: Linux/dbus/wpa_interface.xml
+	gdbus-codegen --header --output $@ --c-namespace Wpa $<
+
+Linux/gen/DBusWpaNetwork.h: Linux/dbus/wpa_network.xml
+	gdbus-codegen --header --output $@ --c-namespace Wpa $<
+
+Linux/gen/DBusWpa.c: Linux/dbus/wpa.xml
+	gdbus-codegen --body --output $@ --c-namespace Wpa $<
+	sed -i 's/config\.h/BuildConfig.h/g' $@
+
+Linux/gen/DBusWpaInterface.c: Linux/dbus/wpa_interface.xml
+	gdbus-codegen --body --output $@ --c-namespace Wpa $<
+	sed -i 's/config\.h/BuildConfig.h/g' $@
+
+Linux/gen/DBusWpaNetwork.c: Linux/dbus/wpa_network.xml
+	gdbus-codegen --body --output $@ --c-namespace Wpa $<
+	sed -i 's/config\.h/BuildConfig.h/g' $@
+
+if CONFIG_WIFI_PLATFORM_WPA
+BUILT_SOURCES                          += \
+    Linux/gen/DBusWpa.h                   \
+    Linux/gen/DBusWpaInterface.h          \
+    Linux/gen/DBusWpaNetwork.h            \
+    Linux/gen/DBusWpa.c                   \
+    Linux/gen/DBusWpaInterface.c          \
+    Linux/gen/DBusWpaNetwork.c            \
```

Comment:
Could instructions for the process for generating these sources from the xml be added to the `Linux/README.md`?

Reply:
Ack, will do after we finalize the WiFi support

Reply:
Filed this: https://github.com/project-chip/connectedhomeip/issues/1936